### ISSUE TITLE
The menu is missing

### ISSUE THREAD

**Issue by ntvuong1709it** [dotnet/docfx#2820]
Hi there,

I just install docfx to VS2017 from Nuget Package. Then I build the project and open the index.html file
However menu is missing.

![image](https://user-images.githubusercontent.com/30897181/40638861-f6a817ce-6337-11e8-907a-ee47e512a367.png)

This is my toc.yml file
- name: Articles
  href: articles/
- name: API Documentation
  href: obj/api/
  homepage: api/index.md

Did I missing something ?
Please help

**Comment by vicancy** [comment]
Are you opening the html directly? navbar is dynamically loaded and needs a web server. Try `docfx build --serve` or host these html files using some web server like https://www.npmjs.com/package/http-server

**Comment by ntvuong1709it** [comment]
Thanks @vicancy ,
It's working fine if host these file on a server rather than just run index.html file

**Comment by bapcyk** [comment]
this works without server in Edge web browser. And does not work with Opera, FF

**Comment by pleonex** [comment]
Just leaving this comment here in case someone needs the menu with static HTML without a web server (for instance, to ship the docs). As said in [this comment](https://github.com/dotnet/docfx/issues/1762#issuecomment-308304237) from #1762, you just need to change the template to `statictoc`.

**Comment by TAGC** [comment]
I was having the same issue with the menu missing. None of the above solutions resolved it. Instead, I resolved this by moving my `docfx.json` file alongside my other documentation files.

Originally, I had `docfx.json` located in the root solution folder which referenced documentation files under `~/resources/docs/`. Specifically, the file layout was:

- `docfx.json`
- `resources/docs/index.md` (a copy of the README.md in the root folder - in a CI pipeline I can create a step that will automatically copy `~/README.md` to `~/resources/docs/index.md`)
- `resources/docs/api/...` (the automatically generated API documentation)
- `resources/docs/toc.yml`

The `resources/docs/toc.yml` file contained:

```yml
- name: Home
  href: index.md
- name: Api Documentation
  href: api/
  ```
  
After moving the `docfx.json` file under `resources/docs` and updating the paths within it accordingly, I found the different items in the TOC file displayed in the navbar and clicking the logo no longer brought me to an invalid location. I guess this makes sense - when I had `docfx.json` in the root solution folder, I was having to serve the website from that directory and it must assume there would be a TOC file at the same level which it uses to populate the navbar.
